cli/remove/remove.go

```diff
@@ -0,0 +1,71 @@
+package remove
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// Remove removes binary/directory and symlinks.
+func Remove(program string, dst string) error {
```

Comment:
What is dst? Is this destination? What do we copy/move to this destination?

Reply:
changed